grue.go

```diff
@@ -29,6 +31,42 @@ func add(args []string, conf *config.GrueConfig) error {
 	return conf.AddAccount(name, uri)
 }
 
+func list(args []string, conf *config.GrueConfig) {
+	const (
+		fmtShort = "%s\t%s\n"
+		fmtFull  = "%s:\n%s\n"
+	)
+	var full bool
+	var listCmd = flag.NewFlagSet("list", flag.ExitOnError)
+	listCmd.BoolVar(&full, "full", false, "Show full account info")
+	listCmd.Parse(args)
```

Comment:
Once you change the FlagSet to ContinueOnError Parse may return an error here, check for it and return it back to main

Reply:
Will change.